Framework/CurveFitting/test/AugmentedLagrangianOptimizerTest.h

```diff
@@ -108,8 +108,8 @@ class AugmentedLagrangianOptimizerTest : public CxxTest::TestSuite {
     auto res = runMinimizer(INEQUALITYCONSTRAINT);
     TS_ASSERT_EQUALS(m_nparams, res.size());
     if (res.size() == m_nparams) {
-      TS_ASSERT_DELTA(1.4709132, res[0], 1e-7);
-      TS_ASSERT_DELTA(0.2941826, res[1], 1e-7);
+      TS_ASSERT_DELTA(1.4709132, res[0], 1e-4);
```

Comment:
I'm a little nervous when things like this change. Do we know why it changed?

Reply:
I spoke to Richard about these changes and it's because of our underlying dependencies we think. I am fairly certain this test is for an algorithm either not in use or won't be soon.